scripts/plot_summary.py

```diff
@@ -379,18 +385,24 @@ def historical_emissions(countries):
     e["waste management"] = "5 - Waste management"
     e["other"] = "6 - Other Sector"
     e["indirect"] = "ind_CO2 - Indirect CO2"
-    e["total wL"] = "Total (with LULUCF)"
-    e["total woL"] = "Total (without LULUCF)"
+    # e["total wL"] = "Total (with LULUCF)"
+    # e["total woL"] = "Total (without LULUCF)"
```

Comment:
this is intended?

Reply:
we can remove it for now. They changed the format in a newer version of the input data and then these two will be needed but I can remove it for now. We can update the data (which will effect also other scripts) in a separate PR.